provider/aws/dist/rack.json

```diff
@@ -960,6 +961,7 @@
     },
     "Nat0": {
       "Condition": "Private",
+      "DependsOn": "KernelUser",
```

Comment:
Should this be `KernelAccess` instead so the creds are generated? It has an implicit dependency on `KernelUser`